components/benefit_sponsors/app/domain/benefit_sponsors/operations/enrollment_eligibility/determine.rb

```diff
@@ -70,21 +70,21 @@ def find_benefit_sponsorship(benefit_sponsorship_entity_id)
         def is_initial_sponsor?(benefit_applications, effective_date)
           recent_benefit_application = benefit_applications.max_by { |benefit_application|  benefit_application.earliest_benefit_application_item.effective_period.min }
           return true unless recent_benefit_application
-          return true if recent_benefit_application.is_termed_or_ineligible? || recent_benefit_application.aasm_state == :active && recent_benefit_application.effective_period.cover?(effective_date)
+          return true if recent_benefit_application.is_termed_or_ineligible? || recent_benefit_application.aasm_state == :active && recent_benefit_application.earliest_benefit_application_item.effective_period.cover?(effective_date)
```

Comment:
as we are considering recent_benefit_application is it need to be latest_benefit_application_item? 

```
recent_benefit_application. latest_benefit_application_item.effective_period.cover?(effective_date)
```